### PR TITLE
Add GitHub Pages Website

### LINKED ISSUE #103: Website for AndrOBD
It would be nice to have an AndrOBD website so any articles in the future can link to it and the website would make a better visual representation of the project that could attract developers/contributors better.

A completely free and simple option is via Github Pages, that a lot of github open source projects are using, and you can even use/buy a custom domain like "www.androbd.org" if you want: https://pages.github.com/

> Hosted directly from your GitHub repository. Just edit, push, and your changes are live.
> No databases are needed to be set up and no servers to configure

Examples of repos that use github pages: https://github.com/collections/github-pages-examples

Interesting article: https://www.toptal.com/github/unlimited-scale-web-hosting-github-pages-cloudflare

> Consider [Jekyll](https://jekyllrb.com/). You give it [Liquid ](https://jekyllrb.com/docs/liquid/)templates and Markdown content
Free Jekyll templates: https://jekyllthemes.io/free

### PR DESCRIPTION
**Pull request by sampoder**
Hi!

This PR will resolve #103.

I have created a static site that can be used as a landing page. It contains basic info about the project. You can view a preview here: https://sampoder.github.io/AndrOBD/.

There are still some improvements but this a good thing to base the site off.

Also, this shouldn't be merged to the master branch, a new branch called gh-pages should be made.

Feedback Appreciated!

Thanks,

 Sam

### CODE REVIEW

**Comment by sampoder** [comment]
Hi! Sorry, this probably should be merged into a seperate gh-pages branch. However this doesn't exist so I was wondering if you could create a gh-pages and then i can change the target branch

**Comment by sampoder** [comment]
I can create a /docs folder but the website is more of a landing page IMO

**Comment by fr3ts0n** [comment]
I have created a new branch "github-pages".
Pls redirect your PR to this branch. I will then revert all your file removals before merging to master ...

**Comment by fr3ts0n** [comment]
Could some graphics / artwork be used (i.e.: from logo subdir)?

**Comment by fr3ts0n** [comment]
Does this implementation interfere with README.md from repo?

**Comment by sampoder** [comment]
> Could some graphics / artwork be used (i.e.: from logo subdir)?

yep sure!

**Comment by sampoder** [comment]
> Does this implementation interfere with README.md from repo?

it shouldn't, for github pages normally you have a completly seperate branch (gh-pages) which it will detect and then your website will be there

**Comment by sampoder** [comment]
@fr3ts0n I have updated it to add the logo!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+                        </div>
+                    </div>
+                </section>
+                
+               
+                <footer class="footer-section mtxl">
+                    <div class="wrapper w960p flex-container flex-container-v txtcenter">
+                         
+                        <div class="h5-like pbs">Built by fr3ts0n • Hosted on <a href="https://github.com/fr3ts0n/AndrOBD">GitHub</a></div>
+                        <div class="h5-like pbl"><a href="https://github.com/fr3ts0n/AndrOBD-Plugin/blob/master/LICENSE"> Licensed under GNU General Public License v3.0</a></div>
```
Comment (fr3ts0n) [inline index.html]: Link is formally incorrect, since it links to licence of plugin repo

Comment (sampoder) [reply index.html]: Sorry! Let me fix this

Comment (sampoder) [reply index.html]: I've fixed this

**Comment by sampoder** [comment]
Awesome!

**Comment by fr3ts0n** [comment]
Sorry I still can not merge it, since it shows the conflict with translation changes. Please could you resolve this?

**Comment by sampoder** [comment]
Fixed!

**Comment by sampoder** [comment]
I'm just a little worried, I'm not sure `github-pages` as a branch name works for the system, I think it has to be `gh-pages`. That seems to be what this: https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site is saying

**Comment by aha999** [comment]
Landing page is awesome, thanks @sampoder !

**Comment by aha999** [comment]
@sampoder that looks really good! You said i should review it...

- I think you should add some screenshots of the interface, and if in the future the GUI gets improved, you can simply replace the existing screenshots there or if they're auto synced...
- typo on the landing page: "fincailly" instead of financially.
- typo: "started fr3ts0n" instead of "started **_by_** fr3ts0n"
- I think donation buttons should look like buttons and be more accessible/visible...
- i think that FAQ questions could be one line above their answers... Like:

> Do you have a chat I can join? 
> Yes, we do. For general discussions, please join this Telegram chat. For updates on test releases, please join this Telegram chat.
>

**Pull request merged**